tests/client.py

```diff
@@ -893,23 +907,50 @@ def create_contact_lists(self):
             using different filters would result in any new fields.
         """
         record_uuid = str(uuid.uuid4()).replace('-', '')
+        value = f"@hubspot{record_uuid}"
 
         url = f"{BASE_URL}/contacts/v1/lists/"
         data = {
             "name": f"tweeters{record_uuid}",
-            "dynamic": True,
+            "dynamic": dynamic,
             "filters": [
                 [{
                     "operator": "EQ",
-                    "value": f"@hubspot{record_uuid}",
+                    "value": value,
                     "property": "twitterhandle",
                     "type": "string"
                 }]
             ]
         }
         # generate a record
         response = self.post(url, data)
+        self.list_id = response.get("listId")
```

Comment:
The list_id is in the response you are returning, there is no need to store it separately

Reply:
Done. Making use of the list_id from response